src/Snap/Internal/Util/FileUploads.hs

```diff
@@ -198,19 +290,20 @@ type PartProcessor a = PartInfo -> InputStream ByteString -> IO a
 -- /Exceptions/
 --
 -- If the given 'UploadPolicy' stipulates that you wish form inputs to be
--- placed in the 'rqParams' parameter map (using 'setProcessFormInputs'), and
--- a form input exceeds the maximum allowable size, this function will throw a
--- 'PolicyViolationException'.
+-- processed (using 'setProcessFormInputs'), and form a form input exceeds
+-- the maximum allowable size or form exceeds maximum number of inputs, this
+-- function will throw a 'PolicyViolationException'.
 --
 -- If an uploaded part contains MIME headers longer than a fixed internal
 -- threshold (currently 32KB), this function will throw a 'BadPartException'.
 --
-handleMultipart ::
+foldMultipart ::
```

Comment:
Can you also please add a "Since" entry for this function? I.e.

```haskell
-- /Since: 1.0.2.0/
```

You'll also need to bump the minor version number to 1.0.2.0 in the .cabal.